docs/faq.md

```diff
@@ -0,0 +1,68 @@
+# F.A.Q.
+
+Here are a couple of common questions about JupyterLab and the transition from
+the classic Notebook interface.
+
+## Is the classic notebook going away ?
+
+No, at least not yet, it will though get way less updates. There will be a
+transition period from Classic notebook to JupyterLab. Currently the notebook
+server you get when you install Jupyter comes with Classic Notebook
+preinstalled. JupyterLab is an extension. When you install JupyterLab, the
+notebook server runs both classic notebook and JupyterLab. 
+
+When JupyterLab has been widely adopted, the server will come with Lab by
+default, and the classic notebook will be an extension. 
+
+## What will happen to my notebook files ?
```

Comment:
I would take change this question to be in the present tense since this is probably an immediate concern for those starting to use JupyterLab e.g., 

Can I still use my notebook files? or What about my notebook files? or What happens to my notebook files?